packages/asc-ui/src/components/InfoCard/InfoCard.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect, useState } from 'react'
+import { Close } from '@datapunt/asc-assets'
+import InfoCardStyle, {
+  Props as StyleProps,
+  CloseButton,
+  CloseButtonWrapper,
+  ContentWrapper,
+} from './InfoCardStyle'
+import Heading from '../Heading'
+import CompactThemeProvider from '../../theme/CompactThemeProvider'
+import Paragraph from '../Paragraph'
+import Link from '../Link'
+import { ButtonVariants } from '../Button/ButtonStyle'
+
+type Props = {
+  heading?: string
+  onClose?: () => void
```

Comment:
Small thing, since we aren't really closing messages but dismissing them perhaps we should use `dismissable` and `onDismiss` instead?

Reply:
Will do!